news/2017-09-21-beatmap-spotlights-july-august-2017.md

```diff
@@ -0,0 +1,259 @@
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/something-just-like-this.jpg)](http://osu.ppy.sh/s/611301)
+
+On his map of this catchy electronic pop piece, [handsome](https://osu.ppy.sh/u/2123087) uses various interesting techniques to push the song to its limits. Extreme contrasts in slider velocity and spacing as well as effective under- and overmapping are used to highlight the varying intensity and tone of the music and also make the map a true challenge to master.
+
+The Insane-level guest difficulties are also a blast to play and feature various creative concepts - sticking out the most are the stunning visuals and beautiful sliderart in Regou's Insane and the unique patterning in fanzhen's Insane.
+
+Since the lower difficulties also mirror the gimmicks of the higher ones in more tame ways, the whole mapset is worth checking out for any osu! player who is interested in experiencing some fresh and creative mapping!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/pop-it.jpg)](http://osu.ppy.sh/s/501677)
+
+This map's special patterning is what makes it stand out. Everything is perfectly aligned to the grid's axes and combined with expressive jump usage and unusual rhythms here and there, unique visuals and gameplay are created.
+
+The same style is used for every difficulty, which makes the mapset feel very coherent and professional. Players who are interested in playing something different from the usual mapping meta should be checking this one out right now!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/dreadnought.jpg)](https://osu.ppy.sh/s/361306)
+
+[Cherry Blossom](https://osu.ppy.sh/u/1156742) has yet again managed to create a map which is not only insanely clean but also very fun to play in general, with an absolutely clever use of symmetry and a very fitting use of spacing for the streams, underlining properly the intensity of the various instruments. If you like challenging stream maps, you can't go wrong with this one!
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/true-blue.jpg)](https://osu.ppy.sh/s/643803)
+
+Wait, what year it is? Eurobeat in 2017?! This time we have a remix of the song True Blue mapped by [iloveyou4ever](https://osu.ppy.sh/u/4964596), he previously mapped the original song and got in the spotlights too, but let's talk about this beautiful map (and song). We will see a good follow of the vocals and instrumental as well some piano, even if the map is 164 bpm you will enjoy it with a lot of streams and some high slider velocity.
+
+The song and the map itself it feels relaxing to play even with the amount of notes it has, as it contains a good structure and a consistent mapping, in my opinion, it's the kind of map that even if you manage to get full combo, you are gonna still replaying it to get a good score on it, and also, don't forget to give it a chance with HardRock and Hidden!
```

Comment:
I think this line needs slight rewording. 
"The song and the map themselves feel relaxing to play even with the high amount of notes it has, as it contains a well structured and consistent mapping in my opinion. It's the kind of map that you'd still replay even after managing to get a full combo on it! Also, don't forget to give it a chance with mods like HardRock or Hidden!"